content/v2.0/monitor-alert/send-email.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Send alert email
+description: >
+  Send an alert email.
+menu:
+  v2_0:
+    parent: Monitor & alert
+weight: 104
+v2.0/tags: [alert, email, notifications, check]
+related:
+  - /v2.0/monitor-alert/checks/
+---
+
+Send an alert email using a third party service, such as [SendGrid](https://sendgrid.com/), [Amazon Simple Email Service (SES)](https://aws.amazon.com/ses/), [Mailjet](https://www.mailjet.com/), or [Mailgun](https://www.mailgun.com/). To send an alert email, complete the following steps:
+
+1. [Create a check](/v2.0/monitor-alert/checks/create/#create-a-check-in-the-influxdb-ui) to identify the data to monitor and the status to alert on.
+2. Set up your preferred email service (sign up, retrieve API credentials, and send test email):
+   - **SendGrid**: See [Getting Started With the SendGrid API](https://sendgrid.com/docs/API_Reference/api_getting_started.html)
+   - **AWS Simple Email Service (SES)**: See [Using the Amazon SES API](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/send-email.html). Your AWS SES request, including the `url` (endpoint), authentication, and the structure of the request may vary. For more information, see [Amazon SES API requests](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/using-ses-api-requests.html) and [Authenticating requests to the Amazon SES API](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/using-ses-api-authentication.html).
+   - **Mailjet**: See [Getting Started](https://dev.mailjet.com/email/guides/getting-started/)
```

Comment:
- Include "with Mailjet" in the link text to indicate the link will take them out of the docs:

```
- **Mailjet**: See [Getting Started with Mailjet](https:...
```

Reply:
good point